tests/org.eclipse.xtend.core.tests/src/org/eclipse/xtend/core/tests/validation/ValidationBug427432Test.xtend

```diff
@@ -41,7 +41,7 @@ class ValidationBug427432Test extends AbstractXtendTestCase {
 				}
 			
 			}
-		'''.parse.assertError(XbasePackage.Literals.XFEATURE_CALL, IssueCodes.DUPLICATE_CASE)
+		'''.parse.assertNoErrors
```

Comment:
I just adapted the expectation to other similar cases. The validation uses `SwitchConstantExpressionsInterpreter`, which is the  point where I fixed the compiler bug. When that interpreter is unable to evaluate the switch cases, the validation simply does nothing.